### PR TITLE
Adds TG vibe

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

Copper no longer is fucked up, medkits, bruise packs and ointments

## Why It's Good For The Game

Better sprites = better look of the game

## Changelog
:cl:
imageadd: Resprited copper, medkits and med shit
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Before people start screeching, no this doesn't change the items mechanically it's just a re-name and a different sprite.

File: code/game/objects/items/stacks/medical.dm [hunk 1]
```diff
@@ -101,10 +101,10 @@
 	use(1)
 
 /obj/item/stack/medical/bruise_pack
-	name = "bruise pack"
-	singular_name = "bruise pack"
+	name = "suture"
+	singular_name = "suture"
 	desc = "A therapeutic gel pack and bandages designed to treat blunt-force trauma."
```
Comment (ghost) [inline code/game/objects/items/stacks/medical.dm:106]:
```suggestion
	desc = "A pack of sutures used to treat blunt-force trauma."
```

**Comment by PowerfulBacon** [comment]
I'm going to screech anyway because I prefer the bee ones (not that they are better I just like the old things)

**Comment by ToasterBan** [comment]
sutures suck ass, tg sucks ass, this pr sucks ass

**Comment by ghost** [comment]
> sutures suck ass, tg sucks ass, this pr sucks ass

You suck ass.

**Comment by ghost** [comment]
> sutures suck ass, tg sucks ass, this pr sucks ass

You suck loadsa ass

**Comment by SomeAngryMiner** [comment]
you added a lot of unused icons (satchels of holding, bags of holding), removed a few things that are still in use, like some surgery kits and other sprites without checking.
Im almost sure you just copied and pasted the tg dmi or something, but if you fix the miriad of conflicts this changes, its okay, i suppose.

**Comment by MNarath1** [comment]
Why does this edit 3 dmi files??? If have not viewed the dmi files yet trough i am pretty sure this should only need the edit of a single dmi file not three of them

**Comment by MNarath1** [comment]
I personally do not really like the new medkits they look like you could smash someones face in with them trough thats only my opinion trough pretty sure this would call for a vote honestly if people actually want those resprites trough we will see if there is going to be a vote

**Comment by MNarath1** [comment]
But even if there is a vote remove the unused icons that this adds if someone wants to port those items they will do so

**Comment by St0rmC4st3r** [comment]
No. I absolutely hate the medkit change. TG medkits are ugly. They look like those soft kindergarten doctor playset boxes. The wrapped boxes look bad and inconsistent with crate geometry. Lack of perspective doesn't help either.
I think that the only sprite from tg that I hated more than these was the new medanalyzer which looked as if the picture was taken from left-bottom of the thing.

**Comment by St0rmC4st3r** [comment]
Can we have actual satures instead?

**Comment by MNarath1** [comment]
> Can we have actual satures instead?

would that not need the port of some of the medical changes of tg?

**Comment by ghost** [comment]
> > Can we have actual satures instead?
> 
> would that not need the port of some of the medical changes of tg?

Yeah, and?

**Comment by St0rmC4st3r** [comment]
> > Can we have actual satures instead?
> 
> would that not need the port of some of the medical changes of tg?

The satures are one of the few /tg/ med changes I liked. It's literally one file.

**Comment by Fronsis** [comment]
I don't like the word suture that's all i have to say.

TG medbay bad

**Comment by knacker48** [comment]
Somehow TG even managed to nerf its sprites. Please don't add these unless they're their own separate item

**Comment by Johnathon-Baxter** [comment]
Having the regenerative meshes and sutures would be nice for the purposes of making it so that you don't constantly have to switch targeting zones.

**Comment by ghost** [comment]
> Having the regenerative meshes and sutures would be nice for the purposes of making it so that you don't constantly have to switch targeting zones.

My idea is to replace all the normal first aid kits with sutures and regen meshes but keep bruise packs and ointments for doctors.

**Comment by Bastian0930** [comment]
anyone got pictures? i'm not booting up dream maker rn

**Comment by ghost** [comment]
> anyone got pictures? i'm not booting up dream maker rn

https://github.com/BeeStation/BeeStation-Hornet/pull/2863/checks?check_run_id=1337869627

**Comment by Bastian0930** [comment]
thank you

**Pull request closed without merging**